src/components/ResourceSettingsModal.jsx

```diff
@@ -246,7 +247,16 @@ export default class ResourceSettingsModal extends Component {
               <p className={elementStyles.formLabel}>Resource Type</p>
 
               {/* Permalink or File URL */}
-              <button type="button" onClick={this.permalinkFileUrlToggle}>{permalink ? 'Switch to download' : 'Switch to post'}</button>
+              <div className="d-flex">
+                <label htmlFor="radio-post" className="flex-fill">
+                  <input type="radio" id="radio-post" name="resource-type" value="post" onClick={this.handlePermalinkFileUrlToggle} checked={!!permalink} />
+                  Post Content
+                </label>
+                <label htmlFor="radio-post" className="flex-fill">
+                  <input type="radio" id="radio-file" name="resource-type" value="file" onClick={this.handlePermalinkFileUrlToggle} checked={!permalink} />
+                  Downloadable File
+                </label>
+              </div>
               {permalink
```

Comment:
We can have more sophisticated logic here (maybe in a future PR) so that we can store the state of the permalink instead of resetting it to null every time we toggle the radio button